app/repositories/formatters/ApprovalProcessFormatter.scala

```diff
@@ -16,21 +16,25 @@
 
 package repositories.formatters
 
-import models.ApprovalProcess
+import models.{ApprovalProcess, ApprovalProcessMeta}
 import play.api.libs.json._
 
 object ApprovalProcessFormatter {
 
+  implicit val metaFormatter: Format[ApprovalProcessMeta] = ApprovalProcessMetaFormatter.mongoFormat
+
   val read: JsValue => JsResult[ApprovalProcess] = json =>
     for {
-      id <- (json \ "_id").validate[String]
+      id <- (json \ "_id").validateOpt[String]
+      meta <- (json \ "meta").validate[ApprovalProcessMeta]
       process <- (json \ "process").validate[JsObject]
-    } yield ApprovalProcess(id, process)
+    } yield ApprovalProcess(id.getOrElse(meta.id), meta, process)
 
   val write: ApprovalProcess => JsObject = approvalProcess =>
     Json.obj(
       "_id" -> approvalProcess.id,
-      "process" -> approvalProcess.process
+      "meta" -> Json.toJsFieldJsValueWrapper(approvalProcess.meta),
```

Comment:
have seen this before, why choose this over ususal object syntax?

Reply:
Good spot - I think this was an intellij autocomplete oversight!  Have changed to use toJson and all tests work as expected